Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 134 paragraph content type

## Related ticket:
[SCRUM-134](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-134?atlOrigin=eyJpIjoiNmEyY2Q5ZjQxOTE2NGU4ZGIyMWMxMDM2YjExYWM4MGIiLCJwIjoiaiJ9)

## In this PR:
- Change content type of front-page to paragraph
- Add 4 paragraphs type for the front-page
- Add content for front-page
- Install REST Entity Recursive and configure the module
- Fetch frontpage content and paragraph content with react
- Apply the fetched data and render the front-page

## Testing instruction
- Checkout branch
- Reinstall backend dependencies: ```lando composer i```
- Import backend db `lando db-import 1.2.gz`
- Import backend config `lando drush cim`
- Check frontage

## Screenshot
Frontpage
![image](https://github.com/user-attachments/assets/d6376197-c279-40c7-804f-0c87725cee18)

Data structure of frontpage endpoint
![image](https://github.com/user-attachments/assets/5aa6dcf0-d590-4597-a9e8-fb8a108fc069)



[SCRUM-134]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-134?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
